certified-connectors/Yeeflow/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "18e3f376-5cfd-1ed1-770f-ba933a6041d1",
```

Comment:
The clientId should be replaced with a dummy value, or `null` for the purposes of open sourcing. You will be able to provide it when submitting the connector for certification.